aimmo-game/simulation/map_generator.py

```diff
@@ -48,15 +48,16 @@ def get_map(self):
         width = self.settings['START_WIDTH']
         world_map = WorldMap.generate_empty_map(height, width, self.settings)
 
-        # We designate one non-corner edge cell as empty, to ensure that the map can be expanded
+        # We set one non-corner edge cell as empty, to ensure that the map can be expanded
         always_empty_edge_x, always_empty_edge_y = get_random_edge_index(world_map)
         always_empty_location = Location(always_empty_edge_x, always_empty_edge_y)
 
         for cell in shuffled(world_map.all_cells()):
-            if cell.location != always_empty_location and random.random() < self.settings['OBSTACLE_RATIO']:
+            if cell.location != always_empty_location and\
```

Comment:
In my code I have always put the `and` on the new line, is there a python convention for where it should go?

Reply:
`and` stays on the first line. It was changed in 2016, I seen it in the changelogs of PEP8 today. Also, their official example guideline class, `Rectangle` shows this:

http://pymbook.readthedocs.io/en/latest/pep8.html

Reply:
https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator

Reply:
Found it! https://hg.python.org/peps/log/tip/pep-0008.txt

More specifically: https://hg.python.org/peps/rev/502239ce4889

Reply:
But we should avoid **line breaks** unless required, and instead use parenthesis for stuff like this to split it over few lines... I'll change it!

Reply:
https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator